apps/website/src/components/Card.astro

```diff
@@ -0,0 +1,81 @@
+---
+import React from 'react';
+import ComponentIcon from './ComponentIcon.astro';
+
+export type Props = {
+  url: string;
+  title: React.ReactNode;
+  description?: React.ReactNode;
+  image?: string;
+  className?: string;
+};
+
+const { url, title, description, image, className } = Astro.props;
+---
+
+<li class={`card ${className}`}>
```

Comment:
astro has [`class:list`](https://docs.astro.build/en/reference/directives-reference/#classlist) which makes this cleaner. also apply rest props

```suggestion
<li class:list={['card', className]} {...rest}>
```